lib/matplotlib/ticker.py

```diff
@@ -2792,33 +2792,42 @@ def ideal_ticks(x):
         return MaxNLocator.tick_values(self, vmin, vmax)
 
     def nonsingular(self, vmin, vmax):
-        initial_range = (1e-7, 1 - 1e-7)
-        if not np.isfinite(vmin) or not np.isfinite(vmax):
-            return initial_range  # no data plotted yet
-
+        standard_minpos = 1e-7
+        initial_range = (standard_minpos, 1 - standard_minpos)
+        swap_vlims = False
```

Comment:
As a state flag, I would rather name this `vlims_swapped`.

Reply:
I agree. But I take the logic (and the name for variables) from `LogLocator`.

What is the best?

 - use `vlims_swapped` in `LogitLocator`,
 - use `swap_vlims` in `LogitLocator` for consistency with `LogLocator`,
 - use `vlims_swapped` in `LogitLocator`, and rename `swap_vlims` to `vlims_swapped` in `LogLocator` for consistency,

I dislike the third one (as it can introduce conflicts with other branches for no real reason). I have no opinion between the two firsts.